gravitee-am-plugins-handlers/gravitee-am-plugins-handlers-api/src/main/java/io/gravitee/am/plugins/handlers/api/plugin/AMPluginHandler.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.plugins.handlers.api.plugin;
+
+import io.gravitee.am.plugins.handlers.api.core.AMPluginManager;
+import io.gravitee.plugin.core.api.AbstractPluginHandler;
+import io.gravitee.plugin.core.api.Plugin;
+import io.gravitee.plugin.core.api.PluginClassLoaderFactory;
+import org.slf4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.Assert;
+
+/**
+ * @author Rémi SULTAN (remi.sultan at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public abstract class AMPluginHandler<T> extends AbstractPluginHandler {
```

Comment:
I know this is quite common in the codebase but I would prefer us to stick to the Java convention of camel casing only the first letter of acronyms, e.g. `HttpServletRequest`. I know even in the jdk there are exceptions but this is not the rule.